gcp/createTerraformAdminProject.sh

```diff
@@ -0,0 +1,48 @@
+# Source: https://cloud.google.com/community/tutorials/managing-gcp-projects-with-terraform
+
+# YOUR_ORG_ID #$ gcloud organizations list
+export TF_VAR_org_id="YOUR_ORG_ID"
```

Comment:
Can we pass this values as arguments when running the script? If we do that, we don't need to change this file when cloning the repo.